Mk3

PropsnWings opened the pull request.

akukan committed:
MK2S update

Updated parts for MK2S update

akukan committed:
MK2S update

Updated parts for MK2S

akukan committed:
Delete LCD_SupportA.stl

akukan committed:
Delete LCD_SupportB.stl

akukan committed:
Delete LCD_cover.stl

akukan committed:
Delete RAMBo-cover.stl

akukan committed:
Delete RAMBo-cover-base.stl

akukan committed:
Delete RAMBo-cover-door.stl

akukan committed:
Delete RAMBo-cover-hinges.stl

akukan committed:
Delete RAMBo-cover-base.scad

akukan committed:
Delete RAMBo-cover-door.scad

akukan committed:
Delete RAMBo-cover-hinges.scad

akukan committed:
Rename RAMBo_hinges.scad to RAMBo-cover-hinges.scad

akukan committed:
Rename RAMBo-base.scad to RAMBo-cover-base.scad

akukan committed:
Rename RAMBo-doors.scad to RAMBo-cover-doors.scad

akukan committed:
Update README.md

MK2S

akukan committed:
Frame dxf added

akukan committed:
Adjusted dimensions, removed ventilation holes

akukan committed:
Adjusted dimensions, removed ventilation holes

akukan committed:
PINDA probe mount changed to one screw

akukan committed:
PINDA probe mount changed to one screw

akukan committed:
Delete FRAME_MK2.dxf

akukan committed:
Delete LCD-cable-clip.scad

akukan committed:
Delete LCD-cover-ORIGINAL.scad

akukan committed:
Delete PSU-Y-part.scad

akukan committed:
Delete PSU-cover-NODRILL.scad

akukan committed:
Delete RAMBo-cover-base.scad

akukan committed:
Delete RAMBo-cover-doors.scad

akukan committed:
Delete RAMBo-cover-hinges.scad

akukan committed:
Delete Y-distance.scad

akukan committed:
Delete belt-guide.scad

akukan committed:
Delete cable-holder.scad

akukan committed:
Delete extruder-body.scad

akukan committed:
Delete extruder-cover.scad

akukan committed:
Delete extruder-idler.scad

akukan committed:
Delete extruder-main.scad

akukan committed:
Delete fan-nozzle.scad

akukan committed:
Delete heatbed_cable_cover.scad

akukan committed:
Delete bearing.scad

akukan committed:
Delete lcd-support-A.scad

akukan committed:
Delete lcd-support-B.scad

akukan committed:
Delete polyholes.scad

akukan committed:
Delete spool holdahMK3.dxf

akukan committed:
Delete spool holdahMK3.scad

akukan committed:
Delete x-carriage.scad

akukan committed:
Delete x-end-idler.scad

akukan committed:
Delete x-end-motor.scad

akukan committed:
Delete x-end.scad

akukan committed:
Delete y-belt-holder.scad

akukan committed:
Delete y-corners.scad

akukan committed:
Delete y-idler.scad

akukan committed:
Delete y-motor.scad

akukan committed:
Delete z-axis-bottom.scad

akukan committed:
Delete z-axis-top.scad

akukan committed:
Delete z-screw-cover.scad

akukan committed:
Delete LCD-cable-clip.stl

akukan committed:
Delete LCD-cover-ORIGINAL.stl

akukan committed:
Delete LCD-knob.stl

akukan committed:
Delete PINDA_protector.stl

akukan committed:
Delete PSU-Y-part.stl

akukan committed:
Delete PSU-cover-NODRILL.stl

akukan committed:
Delete RAMBo-base.stl

akukan committed:
Delete RAMBo-doors.stl

akukan committed:
Delete RAMBo_hinges.stl

akukan committed:
Delete Y-distance.stl

akukan committed:
Delete belt-guide.stl

akukan committed:
Delete z-screw-cover.stl

akukan committed:
Delete cable-holder.stl

akukan committed:
Delete extruder-body.stl

akukan committed:
Delete extruder-cover.stl

akukan committed:
Delete extruder-idler.stl

akukan committed:
Delete fan-nozzle.stl

akukan committed:
Delete heatbed_cable_cover.stl

akukan committed:
Delete lcd-support-A.stl

akukan committed:
Delete lcd-support-B.stl

akukan committed:
Delete spool holdahMK3.stl

akukan committed:
Delete x-carriage.stl

akukan committed:
Delete x-end-idler.stl

akukan committed:
Delete x-end-motor.stl

akukan committed:
Delete y-belt-holder.stl

akukan committed:
Delete y-corners.stl

akukan committed:
Delete y-idler.stl

akukan committed:
Delete y-motor.stl

akukan committed:
Delete z-axis-bottom.stl

akukan committed:
Delete z-axis-top.stl

akukan committed:
MK3 - initial

akukan committed:
Delete i3-anti-vib.scad

akukan committed:
MK3 - initial

akukan committed:
Delete i3-anti-vib.stl

akukan committed:
Update README.md

akukan committed:
Delete Heatbed-MK24.brd

akukan committed:
Delete Heatbed-MK24.sch

akukan committed:
Update README.md

akukan committed:
Frame components added

boelle commented:
@akukan 

i had a look at the MK3v8b.dxf

found a few strange things

4 holes are not marked with what size they should be
3 holes are marked as being threaded/zavit M3 but the hole is M3. Threaded M3 need to be M2.5 and then with a thread cut tool made M3

![image](https://user-images.githubusercontent.com/3832239/33605958-0b2482ba-d9bc-11e7-9c96-bbddd751ffb9.png)

akukan committed:
Spool-holder added

akukan committed:
rpi-zero-frame added

akukan committed:
Heatbed cable cover with screw clip

akukan committed:
Heatbed cable cover with screw clip

akukan committed:
Rotated to proper print position

akukan committed:
Rotated to proper print position

akukan committed:
y-rod-holder added

akukan committed:
y-rod-holder added

akukan committed:
removed, replaced by y-rod-holder

akukan committed:
removed, replaced by y-rod-holder

akukan committed:
removed, replaced by y-rod-holder

akukan committed:
removed, replaced by y-rod-holder

akukan committed:
screws and nuts tolerances

akukan committed:
screws and nuts tolerances

akukan committed:
tolerances

akukan committed:
screws & nuts tolerances , rotated to proper print position

akukan committed:
screws & nuts tolerances , rotated to proper print position

akukan committed:
extruder screw clearance, tolerances

akukan committed:
extruder screw clearance, tolerances

akukan committed:
SD card window size & supports

akukan committed:
SD card window size & supports

akukan committed:
motor opening, tolerances

akukan committed:
motor opening, tolerances

akukan committed:
thin wall cut, tolerances

akukan committed:
thin wall cut, tolerances

akukan committed:
belt tensioner, pulley screw fix, trapeziod nut reinforcement, tolerances

akukan committed:
belt tensioner, pulley screw fix, trapeziod nut reinforcement, tolerances

akukan committed:
Belt holder change, tolerances

akukan committed:
Belt holder change, tolerances

akukan committed:
Tolerances, motor opening & clearance

akukan committed:
Tolerances, motor opening & clearance

veverkaondrej committed:
initial version

veverkaondrej committed:
initial version

veverkaondrej committed:
MK2S to MK3 upgrade - Parts list

akukan committed:
Screw mounted cables version

akukan committed:
Screw mounted cables version

akukan committed:
PSU cover added

akukan committed:
PSU cover added

akukan committed:
Older type of heatbed cover without screw holes

akukan committed:
Older type of heatbed cover without screw holes

akukan committed:
Older type of heatbed cover without screw holes

fandasson committed:
Filename changed + content updated

akukan committed:
Heatbreak and print cooling update

Extruder body and cover : Improved cooling of the heatbreak by redesigning intake / outtake of cooling air with additional air feed opening. Improved PTFE tube holder for better flex materials handling.  Nozzle fan :  fan mounted at 45 degree angle to improve air flow, redesigned nozzle.

akukan committed:
Heatbreak and print cooling update

akukan committed:
Angled fan version

akukan committed:
Delete extruder-body-v2.stl

akukan committed:
Delete extruder-cover-v2.stl

akukan committed:
Heatbreak and print cooling update

Heatbreak and print cooling update, removed second unused screw, cables clearance

fandasson committed:
Information about parts updated

See P3-93 in YouTrack

veverkaondrej committed:
Add files via upload

veverkaondrej committed:
general changes

veverkaondrej committed:
new cable clips, raspberry doors, cable management

- new extruder and heatbed cable holders, clips 
- ziptie cable management added
- new popup doors for raspberry pi (raspberry pi cover available)

veverkaondrej committed:
Delete Einsy-base.stl

veverkaondrej committed:
new cable clips, raspberry doors, cable management 

- new extruder and heatbed cable holders, clips 
- ziptie cable management added
- new popup doors for raspberry pi (raspberry pi cover available)

veverkaondrej committed:
Delete Einsy-hinges.stl

veverkaondrej committed:
resize to fit with Einsy_base

veverkaondrej committed:
Delete Extruder-cable-clip.stl

veverkaondrej committed:
extruder cable clip for new Einsy_base

veverkaondrej committed:
Delete Heatbed-cable-clip.stl

veverkaondrej committed:
heatbed cable clip for new Einsy_base

veverkaondrej committed:
Delete Heatbed-cable-clip_8mm.stl

veverkaondrej committed:
Delete Heatbed-cable-clip_for_8mm_sleeve.stl

veverkaondrej committed:
heatbed cable clip for new Einsy_base (8mm sleeve)

veverkaondrej committed:
Delete heatbed-cable-cover-no-screw.stl

veverkaondrej committed:
nut holders edited, sleeve size changed to 6mm

veverkaondrej committed:
Delete heatbed-cable-cover.stl

veverkaondrej committed:
nut holders edited, sleeve size changed to 6mm

veverkaondrej committed:
Delete x-carriage.stl

veverkaondrej committed:
better nylon holder, nut inserts edited

veverkaondrej committed:
Delete x-end-idler.stl

veverkaondrej committed:
improved nut traps, wall thickness expanded

veverkaondrej committed:
Delete x-end-motor.stl

veverkaondrej committed:
improved nut traps

veverkaondrej committed:
Delete y-motor-holder.stl

veverkaondrej committed:
selective infill improved

veverkaondrej committed:
Delete y-rod-holder.stl

veverkaondrej committed:
tolerances changes

veverkaondrej committed:
Delete z-axis-bottom.stl

veverkaondrej committed:
wall thickness chages

veverkaondrej committed:
Delete raspberry_cover.stl

veverkaondrej committed:
raspberry cover for einsy base, iitial versio

veverkaondrej committed:
Delete raspberry_cover.stl

veverkaondrej committed:
initial version

veverkaondrej committed:
new cable clips, raspberry doors, cable management

veverkaondrej committed:
resize to fit with Einsy_base

veverkaondrej committed:
Older type of heatbed cover without screw holes

veverkaondrej committed:
Screw mounted cables version

veverkaondrej committed:
better nylon holder, nut inserts edited

veverkaondrej committed:
improved nut traps, wall thickness expanded

veverkaondrej committed:
improved nut traps

veverkaondrej committed:
improved nut traps

veverkaondrej committed:
improved nut traps

veverkaondrej committed:
tolerances changes

veverkaondrej committed:
selective infill improved

veverkaondrej committed:
Delete y-motor-holder.scad

veverkaondrej committed:
Delete x-end.scad

veverkaondrej committed:
selective infill improved

veverkaondrej committed:
Tolerances

veverkaondrej committed:
Tolerances

veverkaondrej committed:
Delete heatbed-cable-cover-clip-no-screw.stl

veverkaondrej committed:
heatbed cable clip for new Einsy_base

veverkaondrej committed:
heatbed cable clip for new Einsy_base (8mm sleeve)

veverkaondrej committed:
wall thickness chages

veverkaondrej committed:
extruder cable clip for new Einsy_base

veverkaondrej committed:
Older type of heatbed cover without screw holes (8mm sleeve)

veverkaondrej committed:
Older type of heatbed cover without screw holes

veverkaondrej committed:
Screw mounted cables version (8mm sleeve)

veverkaondrej committed:
Delete heatbed-cable-cover_8mm_sleeve.scad

veverkaondrej committed:
Screw mounted cables version (8mm sleeve)

veverkaondrej committed:
Screw mounted cables version (8mm sleeve)

veverkaondrej committed:
Older type of heatbed cover without screw holes (8mm sleeve)

veverkaondrej committed:
Delete heatbed-cable-cover_8mm_sleeve.scad

veverkaondrej committed:
Older type of heatbed cover without screw holes(8mm sleeve)

veverkaondrej committed:
Screw mounted cables version (8mm sleeve)

veverkaondrej committed:
nut holders edited (8mm sleeve)

veverkaondrej committed:
nut holders edited, sleeve size changed to 6mm

veverkaondrej committed:
nut holders edited (8mm sleeve)

fandasson committed:
Change file content (see CONTENT-225 issue)

akukan committed:
Part rotated top proper print position

akukan committed:
plug aligner added

plug aligner added

akukan committed:
Wall thickness adjusted for better printing
